contracts/clients/src/BlsWalletWrapper.ts

```diff
@@ -164,6 +167,11 @@ export default class BlsWalletWrapper {
     return this.blsWalletSigner.sign(operation, this.privateKey);
   }
 
+  /** Sign a message */
+  signMessage(message: string): Signature {
+    return this.blsWalletSigner.signMessage(message, this.privateKey);
+  }
```

Comment:
This seemed appropriate to expose, as someone may need to use it during the wallet recovery/upgrade process.